### ISSUE TITLE
Angular 15

### ISSUE THREAD

**Issue by bastifix** [abishek/fishbone-diagram-angular#5]
Hey, I'm on Angular 15 and the Ishikawa ist working after finally adding 

`import { NgxFishboneDiagramModule } from 'ngx-fishbone-diagram';`
in the angular module. 
Do you plan any updates for future angular versions?
Thanks in advance!

**Comment by abishek** [comment]
Yes. I do have a local build with angular 15. I ll update it in npmjs by the weekend.
